Swift/QtUI/QtChatWindow.cpp

```diff
@@ -942,6 +952,12 @@ void QtChatWindow::setMessageReceiptState(const std::string& id, ChatWindow::Rec
 
 void QtChatWindow::setBookmarkState(RoomBookmarkState bookmarkState) {
 	roomBookmarkState_ = bookmarkState;
+	if (roomBookmarkState_ == RoomNotBookmarked) {
+		bookmarkStar_->setIcon(QIcon("../resources/icons/star-unchecked2.png"));
```

Comment:
Embedded resources should be used here, so it works in a deployed app and doesn't depend on the directory run from.

Or you could use a star indicator like we use in our vCard code, shown here https://github.com/swift/swift/blob/master/Swift/QtUI/QtVCardWidget/QtVCardGeneralField.cpp#L35 .

A yellow start could mean the room is bookmarked; a white star that the room is not bookmarked.